Cargo.toml

```diff
@@ -43,7 +43,7 @@ quick-error = "2.0"
 rayon = { version = "1.5", optional = true }
 # wasm
 wasm-bindgen-rayon = {version = "1.0", features = ["no-bundler"], optional = true}
-wasm-bindgen = {version = "=0.2.78", optional = true}
+wasm-bindgen = {version = "=0.2.79", optional = true}
```

Comment:
The PR says that there needs to be a resolution on a conflict in Cargo.toml. Other than that it's fine by me.